Sources/SwifterSwift/SwiftStdlib/OptionalExtensions.swift

```diff
@@ -91,15 +91,12 @@ public extension Optional {
 public extension Optional where Wrapped: Collection {
     /// SwifterSwift: Check if optional is nil or empty collection.
     var isNilOrEmpty: Bool {
-        guard let collection = self else { return true }
-        return collection.isEmpty
+        return (self == nil || (self?.isEmpty ?? true))
```

Comment:
I think this was the suggestion
```suggestion
        return self?.isEmpty ?? true
```

Reply:
Oh yes, sorry my bad. Thanks for pointing out.